cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -160,6 +160,16 @@ def _use_sample_sheet_from_housekeeper(
                 "would have copied it to sequencing run directory"
             )
             return
+
+        try:
+            if sample_sheet_path.samefile(run_directory_data.sample_sheet_path):
+                LOG.info(
+                    "Sample sheet from Housekeeper already matches the target location. Skipping."
+                )
+                return
+        except FileNotFoundError:
+            LOG.warning("Sample sheet or target path does not exist.")
```

Comment:
This may not be unexpected, so I would change it from warning to info
```suggestion
            LOG.info("Sample sheet in sequencing directory does not exist.")
```

Reply:
Working with this it is a nice to have the actual path or sample sheet that is being looked for... I.e. make the warning an f-string with the path in in it